cronsul

```diff
@@ -39,9 +52,8 @@ esac
 
 command=("$@")
 
-# This will be wrong if cron starts our job really late.
-schedule_date="$(( ($(date '+%s') / period) * period ))"
+current_run="$(date '+%s')"
```

Comment:
Seems like this and line 28 could end up with a situation where jobs are skipped, e.g.:

 - period=60
 - job runs every minute
 - cronsul runs for the first time at time 0
 - cronsul runs again at time 60
 - Now last_run is 0, current_run is 60, so current_run-period=0 and the check at line 28 passes, so we skip this run.

I guess you'd want to set period to be less than the actual period (probably half?) to prevent this.

If that's the intention, can you add some advice to the README section about --period, and perhaps change the default to 30?